src/pretalx/event/models/event.py

```diff
@@ -139,6 +139,7 @@ class urls(EventUrls):
         ical = '{schedule}.ics'
         feed = '{schedule}/feed.xml'
         location = '{schedule}/location'
+        html_export = f'{settings.MEDIA_URL}/{{self.slug}}/schedule_{{self.slug}}.zip'
```

Comment:
Does this work? I think urls should contain single `{}` always, never `{{ }}`

Reply:
Since this is a format-string `{` is handled by the formatter, `{{` becomes `{` and is handled by urlman, which may not be the prettiest thing. That line will probably have to go anyways, because storing the export in media_root leaks it at a point at which it may not be public yet.

Reply:
✅